src/features/location/components/LocationWidgetDesktop.native.test.tsx

```diff
@@ -121,16 +129,36 @@ describe('LocationWidgetDesktop', () => {
         jest.advanceTimersByTime(1000)
       })
 
+      screen.getByText(
```

Comment:
est ce que ce serait pertinent de séparer en deux tests ?

Reply:
oui c'est pertinent, le mot "and" dans le titre d'un teste est en générale un code smell , j'ai fixé